wdio/test/specs/specs/reservoir.js

```diff
@@ -1,108 +1,98 @@
+  })
 
-//Check subheading
-it('Check the subheading of risk display page', async() =>
-    {
-    await expect(riskdisplay_PO.subHeading).exist
-    const subheadingText = await riskdisplay_PO.subHeading
-    console.log('subheading', await subheadingText.getText())
+  // Check subheading
+  it('Check the subheading of risk display page', async () => {
+    await expect(riskdisplayPO.subHeading).exist
+    // TODO: Check this test
+    // const subheadingText = await riskdisplayPO.subHeading
+    // console.log('subheading', await subheadingText.getText())
     await browser.pause(3000)
-    })
+  })
 
-// Check Risk Types and Risk Levels
-it('Check Risk Types', async() => 
-    {
+  // Check Risk Types and Risk Levels
+  it('Check Risk Types', async () => {
     // Read the no. of risk types
-    await expect(riskdisplay_PO.risktypes).exist
-    const riskcountlength = await riskdisplay_PO.risktypes.length
-    console.log('riskcountlength', riskcountlength)
+    await expect(riskdisplayPO.risktypes).exist
+    const riskcountlength = await riskdisplayPO.risktypes.length
+    // TODO: Check this test
+    // console.log('riskcountlength', riskcountlength)
     // take the count of risk types
-    const risktypes = await riskdisplay_PO.risktypes
-    console.log('risktypes', risktypes)
+    const risktypes = await riskdisplayPO.risktypes
+    // console.log('risktypes', risktypes)
 
     // Read the no. of risk level
-    
-    await expect(riskdisplay_PO.riskLevel).exist
-    const risklevel = await riskdisplay_PO.riskLevel
-    for (let countrisk = 0; countrisk < riskcountlength; countrisk++)
-    {
-        console.log('I am in the loop')
-        if (await risktypes[countrisk].getText() == 'Reservoirs') 
-        {     
-            console.log('I could find Rservoir Risk')
 
-                const reservoir = await $('.reservoirs')
-                const htmlStructure = await reservoir.getHTML()
-                console.log('HTML structure:', htmlStructure)
-                const allText = await browser.execute((structure) => 
-                {
-                const tempElement = document.createElement('div')
-                tempElement.innerHTML = structure;
-                let text = tempElement.innerText.trim()
-                text = text.replace(/\s+/g, ' ') // Replace multiple spaces with a single space
-                text = text.replace(/\n+/g, '\n') // Remove consecutive new lines
-                return text
-                }, htmlStructure)
-                console.log('All text:', allText)
-                const reservoirText = riskInfotext.reservoirrisk.toString().replace(/\r\n/g, "\n")
-                console.log('Risk Info text', reservoirText)
+    await expect(riskdisplayPO.riskLevel).exist
+    // const risklevel = await riskdisplayPO.riskLevel
+    await riskdisplayPO.riskLevel
+    for (let countrisk = 0; countrisk < riskcountlength; countrisk++) {
+      // console.log('I am in the loop')

```

Comment:
Can see lots of console.logs commented out. Not sure whether these are here because we plan on coming back to these tests. Happy to approve still but might be worth considering removing to reduce tech debt.